java-chassis-dependencies/default/pom.xml

```diff
@@ -1121,6 +1121,11 @@
         <artifactId>handler-bizkeeper</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
```

Comment:
better add java-chassis-distribution along

Reply:
add to java-chassis-distribution & solution-basic & coverage-reports